website/docs/components/reveal/partials/guidelines/guidelines.md

```diff
@@ -2,34 +2,58 @@
 
 ### When to use
 
-- {description}
+- To hide additional information until it is relevant to users.
 
 ### When not to use
 
-- {description}, consider {[component](#)}.
-- {description}, consider {[component](#)}.
-- {description}, consider {[component](#)}.
+- To hide information critical to the user’s task.
+- To contain complex or advanced content.
 
-### Variant/property name
+## Toggle text
 
-<!-- don’t forget to include real examples and do/don’t blocks, as necessary -->
-{description}
+- Ensure that the text clearly describes the content that will be revealed when the toggle is triggered.
+- Keep the text short, clear, and concise. Avoid long sentences.
+- Use action-oriented language when possible to convey that there is additional content to be shown. e.g. Show more options.
 
-### Variant/property name
+!!! Dont
 
-<!-- don’t forget to include real examples and do/don’t blocks, as necessary -->
-{description}
+Don’t use vague or unrelated text to describe the content.
 
-## Content
+![Reveal example with a form](/assets/components/reveal/toggle-text-dont.png =591x*)
+!!!
 
-- {description}
-- {description}
-- {description}
-- {description}
-- {description}
+!!! Do
+
+Use meaningful text that relates to the content inside.
+
+![Reveal example with a form](/assets/components/reveal/toggle-text-do.png =591x*)
+!!!
+
+## Content type
+
+The Reveal component should be used for hiding and showing simple content. For complex content and advanced functionality, consider using an Accordion (coming soon).
+
+!!! Dont
+
+![Reveal example with complex content](/assets/components/reveal/content-type-dont.png =725x*)
+!!!
+
+!!! Do
+
+![Reveal example with simple content](/assets/components/reveal/content-type-do.png =604x*)
+!!!
+
+## Spacing
```

Comment:
The only feedback I have here is that it seems like this spacing is for a use-case where the reveal is coupled with a specific field; e.g. giving more context/details about the field directly above it. Would the spacing differ if the content within the reveal was related to the entire section of fields?

Reply:
@jorytindall I redid this part to improve the section and cover more details. Hopefully, it's clearer and more descriptive now. I also updated the images to be more generic/agnostic.

One thing to note:
- I found it hard to give a recommendation on the margin-bottom because it depends on what content is below the component. There could be an element that is part of the same group, a separate section. The Reveal could be at the end of a form, followed by button groups. It could be inside a card with nothing else below, etc...